SourceCode/Java/holer-common/src/test/java/org/holer/common/util/HolerConfigTest.java

```diff
@@ -0,0 +1,104 @@
+package org.holer.common.util;
+
+import com.diffblue.deeptestutils.Reflector;
+import com.diffblue.deeptestutils.mock.DTUMemberMatcher;
+import org.apache.commons.lang.StringUtils;
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.powermock.api.mockito.PowerMockito;
+import org.powermock.core.classloader.annotations.PrepareForTest;
+import org.powermock.modules.junit4.PowerMockRunner;
+import java.lang.reflect.Method;
+import java.util.Properties;
+
+import static org.mockito.AdditionalMatchers.or;
+import static org.mockito.Matchers.isA;
+import static org.mockito.Matchers.isNull;
+
+@RunWith(PowerMockRunner.class)
+public class HolerConfigTest {
+
+    @Test
+    public void testStrValue1() {
+        Assert.assertEquals("a\'b\'c", new HolerConfig(null)
+                .getConfig(null).strValue("a\'b\'c", "a\'b\'c"));
+    }
+
+    @PrepareForTest(StringUtils.class)
+    @Test
+    public void testStrValue2() throws Exception {
+        PowerMockito.mockStatic(StringUtils.class);
+        HolerConfig objectUnderTest = (HolerConfig)
+                Reflector.getInstance("org.holer.common.util.HolerConfig");
+
+        Properties properties = new Properties();
+        Reflector.setField(objectUnderTest, "holerConf", properties);
+        Method isBlankMethod = DTUMemberMatcher
+                .method(StringUtils.class, "isBlank", String.class);
+        PowerMockito.doReturn(false)
+                .when(StringUtils.class, isBlankMethod)
+                .withArguments(or(isA(String.class), isNull(String.class)));
+
+        Assert.assertNull(objectUnderTest.strValue("a\'b\'c", "a\'b\'c"));
+    }
+
+    @PrepareForTest(StringUtils.class)
+    @Test
+    public void testIntValue1() throws Exception {
+        PowerMockito.mockStatic(StringUtils.class);
+        HolerConfig objectUnderTest = (HolerConfig)
+                Reflector.getInstance("org.holer.common.util.HolerConfig");
+
+        Properties properties = new Properties();
+        properties.setProperty("\u0000\u0000", "9");
+        Reflector.setField(objectUnderTest, "holerConf", properties);
```

Comment:
Since this project has Apache `commons-lang` as a dependency already, you can avoid using `deeptestutils` by using `FieldUtils.writeField(objectUnderTest, "holerConf", properties, true);` (you can do this for every `Reflector.setField` in all tests). You'll need to import `org.apache.commons.lang.reflect.FieldUtils`.

Reply:
Done